RobotCode/src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -1,105 +1,101 @@
-    }
-
-    public double getPosition()
-    {
-        return INTAKE_RAISER_MOTOR_LEFT.getEncoder().getPosition();
-    }
-
-    public void raiseOrLowerIntakeMount(boolean lowered)
-    {
-        // This is called a ternary operator. Look it up if you don't know what it is.
-        INTAKE_RAISER_MOTOR_RIGHT.set(lowered ? -MOTOR_SPEED : MOTOR_SPEED);
-        isLowered = lowered;
-    }
-
-    // Yes, this is the most descriptive name I could come up with.
-    // The original name beore was "toggleIntakeMotors"
-    // ...which I then changed to "togleIntakeMotorState"
-    // ...which I then changed to this.
-    public void turnOnOrStopIntakeMotors(boolean on)
-    {
-        INTAKE_TOGGLE_MOTOR.set(on ? MOTOR_SPEED : 0);
-    }
-
-    @Override
-    public void periodic()
-    {
-        double difference = goalRotation - getCurrentRotation();
-        setMotors(
-                Math.abs(difference) < DEADBAND ? 0
-                        : Math.max(-1,
-                                Math.min(distance * Math.sin(difference * Math.PI / distance), 1)),
-                "Intake Mount (Deadbanded): Difference: " + difference + ", Distance: " + distance);
-    }
-
-    public void setGoalRotation(double degrees)
-    {
-        distance = degrees - getCurrentRotation();
-        goalRotation = degrees;
-    }
-
-    public double getCurrentRotation()
-    {
-        return INTAKE_RAISER_MOTOR_LEFT.getEncoder().getPosition();
-    }
-
-    public void setMotors(double power, String reason)
-    {
-        INTAKE_RAISER_MOTOR_RIGHT.set(Math.max(-1, Math.min(power, 1)) * MOTOR_SPEED, reason);
-    }
+
+        private TeamSparkMAX intakeDeployMotorLeft, intakeDeployMotorRight, intakeRollerMotorTop,
+                        intakeRollerMotorBottom;
+        private double desiredRotation, desiredVelocity;
+        private SparkPIDController intakeDeployPidController, intakeRollerPidController;
+        private TrapezoidProfile intakeDeployProfile;
+        private RelativeEncoder intakeDeployEncoder;
+
+        public IntakeSubsystem()
+        {
+                intakeDeployMotorRight = new TeamSparkMAX("Intake Deploy Motor Right",
+                                Ports.INTAKE_DEPLOY_MOTOR_RIGHT);
+                intakeDeployMotorLeft = new TeamSparkMAX("Intake Deploy Motor Left",
+                                Ports.INTAKE_DEPLOY_MOTOR_LEFT);
+                intakeRollerMotorTop = new TeamSparkMAX("Intake Roller Motor Top",
+                                Ports.INTAKE_ROLLER_MOTOR_TOP);
+                intakeRollerMotorBottom = new TeamSparkMAX("Intake Roller Motor Bottom",
+                                Ports.INTAKE_ROLLER_MOTOR_BOTTOM);
+
+                // Configure deployment motors
+                intakeDeployMotorLeft.follow(intakeDeployMotorRight, true);
+                intakeDeployMotorRight.enableVoltageCompensation(Constants.MAX_VOLTAGE);
```

Comment:
Unnecessary unless using a voltage other than 12. The motor constructor enables voltage comp with 12 volts.